ice40/utils/ice40_import_bel_timing.py

```diff
@@ -0,0 +1,316 @@
+                pin_map('RADDR[6]', 'RADDR[6]', False),
+                pin_map('RADDR[7]', 'RADDR[7]', False),
+                pin_map('RADDR[8]', 'RADDR[8]', False),
+                pin_map('RADDR[9]', 'RADDR[9]', False),
+                pin_map('RADDR[10]', 'RADDR[10]', False),
+                pin_map('WDATA[0]', 'WDATA[0]', False),
+                pin_map('WDATA[1]', 'WDATA[1]', False),
+                pin_map('WDATA[2]', 'WDATA[2]', False),
+                pin_map('WDATA[3]', 'WDATA[3]', False),
+                pin_map('WDATA[4]', 'WDATA[4]', False),
+                pin_map('WDATA[5]', 'WDATA[5]', False),
+                pin_map('WDATA[6]', 'WDATA[6]', False),
+                pin_map('WDATA[7]', 'WDATA[7]', False),
+                pin_map('WDATA[8]', 'WDATA[8]', False),
+                pin_map('WDATA[9]', 'WDATA[9]', False),
+                pin_map('WDATA[10]', 'WDATA[10]', False),
+                pin_map('WDATA[11]', 'WDATA[11]', False),
+                pin_map('WDATA[12]', 'WDATA[12]', False),
+                pin_map('WDATA[13]', 'WDATA[13]', False),
+                pin_map('WDATA[14]', 'WDATA[14]', False),
+                pin_map('WDATA[15]', 'WDATA[15]', False),
+                pin_map('WCLK', 'WCLK', True),
+                pin_map('WE', 'WE', False),
+                pin_map('WCLKE', 'WCLKE', False),
+                pin_map('WADDR[0]', 'WADDR[0]', False),
+                pin_map('WADDR[1]', 'WADDR[1]', False),
+                pin_map('WADDR[2]', 'WADDR[2]', False),
+                pin_map('WADDR[3]', 'WADDR[3]', False),
+                pin_map('WADDR[4]', 'WADDR[4]', False),
+                pin_map('WADDR[5]', 'WADDR[5]', False),
+                pin_map('WADDR[6]', 'WADDR[6]', False),
+                pin_map('WADDR[7]', 'WADDR[7]', False),
+                pin_map('WADDR[8]', 'WADDR[8]', False),
+                pin_map('WADDR[9]', 'WADDR[9]', False),
+                pin_map('WADDR[10]', 'WADDR[10]', False),
+                pin_map('MASK[0]', 'MASK[0]', False),
+                pin_map('MASK[1]', 'MASK[1]', False),
+                pin_map('MASK[2]', 'MASK[2]', False),
+                pin_map('MASK[3]', 'MASK[3]', False),
+                pin_map('MASK[4]', 'MASK[4]', False),
+                pin_map('MASK[5]', 'MASK[5]', False),
+                pin_map('MASK[6]', 'MASK[6]', False),
+                pin_map('MASK[7]', 'MASK[7]', False),
+                pin_map('MASK[8]', 'MASK[8]', False),
+                pin_map('MASK[9]', 'MASK[9]', False),
+                pin_map('MASK[10]', 'MASK[10]', False),
+                pin_map('MASK[11]', 'MASK[11]', False),
+                pin_map('MASK[12]', 'MASK[12]', False),
+                pin_map('MASK[13]', 'MASK[13]', False),
+                pin_map('MASK[14]', 'MASK[14]', False),
+                pin_map('MASK[15]', 'MASK[15]', False),
+            ]
+        ),
+    'SB_CARRY': ('ICE_CARRY_IN_MUX', [])
+}
+
+
+def get_scale(timescale):
```

Comment:
Should this be part of the python-sdf-parser library? It seems like something every user of that library will need to do?

Reply:
Agree. I'll make a PR to do that.